drivers/hmis/app/graphql/types/hmis_schema/ce_opportunity.rb

```diff
@@ -19,6 +19,7 @@ class HmisSchema::CeOpportunity < Types::BaseObject
     field :eligibility_requirements, [HmisSchema::CeMatchRule], null: true
     field :priority_scheme, HmisSchema::CeMatchRule, null: true
     field :categories, [String], null: false
+    field :swimlanes, [Types::HmisSchema::CeSwimlane], null: true
```

Comment:
Can you explain a little more why we resolve swimlanes on the ce opportunity?
Thinking about the workflow, we assign the swimlanes after the referral is created. So I would think it would make more sense to resolve them directly on the referral object. The participants are associated to the referral, not the opportunity.